apps/scan/frontend/src/components/printer_management/print_test_page_modal.tsx

```diff
@@ -64,13 +87,34 @@ export function PrintTestPageModal({
     );
   }
 
+  if (
```

Comment:
Flow after modifications in this PR:

https://github.com/user-attachments/assets/d184e3e1-5b04-4547-8a96-9fbfa15fb970